### PR TITLE
Added employers and jobs model & migrations.

### PR DESCRIPTION
**Pull request by creative-andrew**

### CODE REVIEW

File: app/Models/Job.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     * @var array<int, string>
+     */
+    protected $fillable = [
+        'keywords',
+        'title',
+        'location',
+        'type'
+    ];
+
+    public function jobs() {
```
Comment (chrisbaltazar) [inline app/Models/Job.php:24]: This method should be names “employer” as the model which refers to

File: app/Models/Job.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        return $this->hasOne(Employer::class);
```
Comment (chrisbaltazar) [inline app/Models/Job.php:25]: If I’m not wrong this is the inverse relationship, which means that must be “belongsTo” instead

File: app/Models/Employer.php [hunk 3]
```diff
@@ -8,7 +8,7 @@
 use Illuminate\Notifications\Notifiable;
 use Laravel\Sanctum\HasApiTokens;
 
-class User extends Authenticatable
+class Employer extends Authenticatable
```
Comment (chrisbaltazar) [inline app/Models/Employer.php:11]: Perhaps you could have also left this model untouched and related it to another one for the Employers, on one on one relationship, up to you

File: app/Models/Job.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+class Job extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $fillable = [
```
Comment (chrisbaltazar) [inline app/Models/Job.php:17]: One tip for the future, you can also just declare the model property “guarded” usually with only the table id field and that means all the rest are allowed to be filled out, at the end the security is implemented by the controllers when validating the request

File: database/migrations/2022_04_29_184818_create_jobs_table.php [hunk 5]
```diff
@@ -13,13 +13,13 @@
...
-            $table->string('name');
-            $table->string('token', 64)->unique();
-            $table->text('abilities')->nullable();
-            $table->timestamp('last_used_at')->nullable();
+            $table->foreignId('employers_id')->references('id')->on('employers')->onUpdate('cascade')->onDelete('cascade');
+            $table->string('keywords');
+            $table->string('title');
+            $table->string('location');
+            $table->string('type');
             $table->timestamps();
```
Comment (chrisbaltazar) [inline database/migrations/2022_04_29_184818_create_jobs_table.php:23]: Have you considered to use “softDeletes” for your models? Is that so you should include the proper field here and the trait on the models, check the docs

**Review by chrisbaltazar — CHANGES_REQUESTED** [review]
Shaping up, things to improve IMO

File: app/Models/Job.php (hunk 1 above)
Comment (creative-andrew) [reply app/Models/Job.php:24]: Done

File: database/migrations/2022_04_29_184818_create_jobs_table.php (hunk 5 above)
Comment (creative-andrew) [reply database/migrations/2022_04_29_184818_create_jobs_table.php:23]: I've read the docs about soft deletes now, thanks for pointing it out, but I don't think I need it at this. point.

**Pull request merged**